src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -117,7 +119,10 @@ private void initialize() {
      */
     @FXML
     private void handleDeletePersons() {
-        final List<ReadOnlyViewablePerson> selected = personListView.getSelectionModel().getSelectedItems();
+        final List<ReadOnlyViewablePerson> selected = new ArrayList<>(personListView.getSelectionModel()
+                                                                                    .getSelectedItems());
+        selected.removeIf(p -> !(p instanceof  ReadOnlyViewablePerson));
```

Comment:
it doesn't work
![image](https://cloud.githubusercontent.com/assets/9070608/16444279/eee096aa-3e0f-11e6-9bfd-9793b2781b26.png)